packages/auth/src/graphql/schema.graphql

```diff
@@ -1,158 +1,185 @@
+  "The cursor used to fetch the next page when paginating forward."
   endCursor: String
-  "Paginating forwards: Are there more pages?"
+  "Indicates if there are more pages when paginating forward."
   hasNextPage: Boolean!
-  "Paginating backwards: Are there more pages?"
+  "Indicates if there are more pages when paginating backward."
   hasPreviousPage: Boolean!
-  "Paginating backwards: the cursor to continue."
+  "The cursor used to fetch the next page when paginating backward."
   startCursor: String
 }
 
 type GrantsConnection {
+  "Information to aid in pagination."
   pageInfo: PageInfo!
+  "A list of edges representing grants and cursors for pagination."
   edges: [GrantEdge!]!
 }
 
 type GrantEdge {
+  "A grant node in the list."
   node: Grant!
+  "A cursor for paginating through the grants."
   cursor: String!
 }
 
 input GrantFilter {
+  "Filter grants by their unique identifier."
   identifier: FilterString
+  "Filter grants by their state."
   state: FilterGrantState
+  "Filter grants by their finalization reason."
   finalizationReason: FilterFinalizationReason
 }
 
 input FilterString {
+  "Array of strings to filter by."
   in: [String!]
 }
 
 input FilterGrantState {
+  "List of states to include in the filter."
   in: [GrantState!]
+  "List of states to exclude in the filter."
   notIn: [GrantState!]
 }
 
 input FilterFinalizationReason {
+  "List of finalization reasons to include in the filter."
   in: [GrantFinalization!]
+  "List of finalization reasons to exclude in the filter."
   notIn: [GrantFinalization!]
 }
 
 input RevokeGrantInput {
+  "Unique identifier of the grant to revoke."
   grantId: String!
 }
 
 interface Model {
+  "Unique identifier for the model."
   id: ID!
+  "The date and time when the model was created."
   createdAt: String!
 }
 
 type Grant implements Model {
-  "Grant id"
+  "Unique identifier of the grant."
   id: ID!
-  "Wallet address of the grantee's account"
+  "Wallet address of the grantee's account."
   client: String!
-  "Access details"
+  "Details of the access provided by the grant."
   access: [Access!]!
-  "State of the grant"
+  "Current state of the grant."
   state: GrantState!
-  "Reason a grant was finalized"
+  "Reason why the grant was finalized."
   finalizationReason: GrantFinalization
-  "Date-time of creation"
+  "The date and time when the grant was created."
   createdAt: String!
 }
 
 type Access implements Model {
-  "Access id"
+  "Unique identifier of the access object."
   id: ID!
-  "Wallet address of a sub-resource (incoming payment, outgoing payment, or quote)"
+  "Wallet address of the sub-resource (incoming payment, outgoing payment, or quote)."
   identifier: String
-  "Access type (incoming payment, outgoing payment, or quote)"
+  "Type of access (incoming payment, outgoing payment, or quote)."
   type: String!
-  "Access action (create, read, list or complete)"
+  "Actions allowed with this access (create, read, list, or complete)."
```

Comment:
```suggestion
  "Actions allowed with this access"
```
There are more than those listed, and in different combination per each resource so safer to not mention I think